### ISSUE TITLE
Uncaught TypeError: undefined is not a function

### ISSUE THREAD

**Issue by poelstra** [TypeStrong/atom-typescript#191]
First of all: thank you very much for Atom Typescript! It truely lives up to its promise of "simply working" out of the box!

I ran into a small issue while trying to rename a class property:

**Atom Version**: 0.187.0
**System**: linux 3.13.0-45-generic
**Thrown From**: [atom-typescript](https://github.com/TypeStrong/atom-typescript) package, v1.1.0

### Stack Trace

Uncaught TypeError: undefined is not a function

```
At /home/martin/.atom/packages/atom-typescript/dist/main/atom/commands.js:107

TypeError: undefined is not a function
  at Object.renameView.panelView.renameThis.onCommit (/home/martin/.atom/packages/atom-typescript/dist/main/atom/commands.js:100:67)
  at atom-text-editor.<anonymous> (/home/martin/.atom/packages/atom-typescript/dist/main/atom/views/renameView.js:29:35)
  at atom-text-editor.jQuery.event.dispatch (/opt/atom/resources/app/node_modules/jquery/dist/jquery.js:4430:9)
  at atom-text-editor.elemData.handle (/opt/atom/resources/app/node_modules/jquery/dist/jquery.js:4116:28)

```

### Commands

```
     -0:20.8 editor:move-to-end-of-screen-line (atom-text-editor.editor)
  5x -0:20.5 core:delete (atom-text-editor.editor)
 11x -0:18.6 editor:move-to-end-of-word (atom-text-editor.editor)
     -0:17.1 editor:move-to-beginning-of-word (atom-text-editor.editor)
     -0:16.7 core:backspace (atom-text-editor.editor)
     -0:16.5 editor:newline (atom-text-editor.editor)
     -0:14.7 editor:move-to-end-of-screen-line (atom-text-editor.editor)
  4x -0:14.0 core:delete (atom-text-editor.editor)
  3x -0:11.2 core:move-down (atom-text-editor.editor)
 29x -0:09.5 core:move-left (atom-text-editor.editor)
  2x -0:07.7 core:move-right (atom-text-editor.editor)
  5x -0:07.2 core:backspace (atom-text-editor.editor)
     -0:05.5 core:undo (atom-text-editor.editor)
     -0:04.6 core:move-left (atom-text-editor.editor)
     -0:03.9 typescript:rename-variable (atom-text-editor.editor)
     -0:00.4 core:confirm (atom-text-editor.editor.mini.is-focused)
```

### Config

```json
{
  "core": {
    "themes": [
      "atom-dark-ui",
      "atom-dark-syntax"
    ],
    "disabledPackages": [
      "metrics"
    ]
  }
}
```

### Installed Packages

```coffee
# User
atom-typescript, v1.1.0
autocomplete-plus, v2.6.0
linter, v0.12.0

# Dev
No dev packages
```

**Comment by basarat** [comment]
Thanks for the kind words!

If you open up a new project with just one class and rename a member of the class, does this happen? If so can I have the class?

**Comment by basarat** [comment]
Having trouble reproducing it with a quick test. Probably something more is going on. Will keep an eye out for it though :rose: 

![classtest](https://cloud.githubusercontent.com/assets/874898/6720710/cfb56cf4-ce19-11e4-8c96-c2c5b2caa2b3.gif)

**Comment by poelstra** [comment]
Thanks for the super fast reply!

I just tried it at work, on Windows using Atom Typescript 1.3.0, and it works like a charm now.
Not sure why it didn't work at home (on Linux, with 1.1.0), I indeed tried renaming another variable in another file, and that didn't work either. Differences are the OS, package version, and the sources (had a fresh checkout at work, in-progress stuff at home).

I'll make sure to check it at home again, but that may take a couple of days.

**Comment by basarat** [comment]
> I'll make sure to check it at home again, but that may take a couple of days.

No hurry. Hopefully these issues will fix themselves as we continue to squash edge :bug: and use it more and more :heart:

**Comment by poelstra** [comment]
Tried it real quick. I still had my laptop hibernated, and the error was still there: couldn't rename any variables. I then just restarted Atom (still Atom Typescript 1.1.0), and the renaming worked again.

I don't know exactly what I did, but I was in the process of moving classes around, thereby making copies of one or two files, then moving the class definitions around in them. Changed definitions from "export class ..." to "export = foo" style, don't know if that's relevant.

I'll keep an eye out for it. If it happens again I'll re-open this issue, but as I cannot reproduce it anymore, I'll close it for now.
